packagehandling/queue-server/queue-server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+require('dotenv').config();
+
+const io = require('socket.io-client');
+const uuid = require('uuid').v4;
+const host = process.env.HOST;
+
+const capsConnect = io.connect(`${host}/caps`);
+console.log(' i am a dragon');
+const messageQueue = {
+  pickup : {},
+
+};
+capsConnect.on('pickup', orderInfo => {
+  console.log('in the pickup recived');
+  const messageId =uuid();
+  const storeName = orderInfo.storeName;
+  messageQueue.pickup['Event'] = 'pickup';
+  messageQueue.pickup[messageId] = orderInfo;
+  messageQueue.pickup['store name'] = storeName;
```

Comment:
When I start this up, I don't get messages in queue-server or driver and then vendor is coming back as an [object Object]